src/main/java/com/coveros/selenified/selenium/Element.java

```diff
@@ -173,7 +178,12 @@ public WebElement getWebElement() {
         if (elements.size() > match) {
             return elements.get(match);
         }
-        return driver.findElement(defineByElement());
+        try {
+            return driver.findElement(defineByElement());
+        } catch (InvalidLocatorTypeException e) {
+            log.error(e);
+            return null;
```

Comment:
if we return null, won't that break other things?
since we can't perform actions on a null element?
Or since this is unreachable code, do we not really care?